Python/Product/Analysis/Parsing/Tokenizer.cs

```diff
@@ -1200,30 +1212,32 @@ public Tokenizer(PythonLanguageVersion version, ErrorSink errorSink = null, Toke
                     case 'L':
                         MarkTokenEnd();
 
-                        if (_langVersion.Is3x()) {
-                            ReportSyntaxError(new IndexSpan(_tokenEndIndex - 1, 1), "invalid token", ErrorCodes.SyntaxError);
+                        if (Verbatim) {
```

Comment:
Is there code somewhere else that will report the syntax error on 3.x (for invalid 'L')?

Reply:
No, I'll add it (and some tests)